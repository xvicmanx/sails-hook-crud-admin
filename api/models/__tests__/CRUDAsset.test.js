import model from '../CRUDAsset';

describe('api::models::CRUDAsset', () => {
  test('The model is defined properly', () => {
    expect(model).toMatchSnapshot();
  });
});
