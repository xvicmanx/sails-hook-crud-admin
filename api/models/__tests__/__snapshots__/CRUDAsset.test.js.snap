// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api::models::CRUDAsset The model is defined properly 1`] = `
Object {
  "attributes": Object {
    "fileDirectory": Object {
      "required": true,
      "type": "string",
    },
    "hash": Object {
      "required": true,
      "type": "string",
      "unique": true,
    },
    "model": Object {
      "required": true,
      "type": "string",
    },
    "name": Object {
      "required": true,
      "type": "string",
    },
    "type": Object {
      "required": true,
      "type": "string",
    },
    "url": Object {
      "type": "string",
    },
  },
}
`;
